File: rBuild/helloworld.R
# Load required libraries
library(ggplot2)

# Print "Hello, World!"
cat("Hello, World!\n")

# Define a vector of numbers
numbers <- c(1, 2, 3, 4, 5)

# Calculate the sum of numbers
total <- sum(numbers)

# Print the sum
cat("Sum of numbers:", total, "\n")

# Create a dataframe for plotting
df <- data.frame(numbers)

# Plot the numbers
ggplot(df, aes(x = numbers)) +
  geom_bar(fill = "blue", stat = "count") +
  labs(title = "Sum of Numbers", x = "Numbers", y = "Count") +
  theme_minimal()

# Save the plot to a file
ggsave("plot.png")

# Save the sum to a file
write(total, file = "sum.txt")

File: jupyterLabBuild/helloworld.py
# Print "Hello!"
print("Hello!")

# Define a list of numbers
numbers = [1, 2, 3, 4, 5]

# Calculate the sum of numbers
total = sum(numbers)

# Save the sum to a file
with open("helloNumbers.txt", "w") as file:
    file.write(f"The sum of numbers is: {total}\n")

print("I just made a file.")